src/AppBundle/DataFixtures/ORM/bluray.yml

```diff
@@ -0,0 +1,19 @@
+AppBundle\Entity\Bluray:
+    ironman:
+        title: Iron Man
+        description: Texte en attente Texte en attente Texte en attente Texte en attente Texte en attente Texte en attente Texte en attente Texte en attente Texte en attente 
```

Comment:
You can use fake description. See https://github.com/hautelook/AliceBundle#basic-usage